src/zkscript/merkle_trees/merkle_tree.py

```diff
@@ -0,0 +1,198 @@
+from tx_engine import Script
```

Comment:
Either make the unlocking script methods static or move them out of the MerkleTree class (they do not make use of self).

Reply:
Actually, it makes little sense to have a class here. We can simply have four functions